services/web/server/tests/integration/01/utils.py

```diff
@@ -0,0 +1,67 @@
+import sys
```

Comment:
We need to talk about these ``utils`` modules in the tests. 
I have been removing them because importing them is unreliable:  import will fail depending on the working directory that pytest has at execution time.

Two options here:

1. Create fixtures that return ``Callables``
2. Add the functions in this module within ``pytest_simcore.helpers`` (try to reuse as much as possible of what is already there, please)

Reply:
Moved both to callabels